### PR TITLE
Logs Panel: Generate types from schema

### PR DESCRIPTION
**Pull request by Clarity-89**
**What is this feature?**

Generate Logs Panel types and use them on front end. 

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: packages/grafana-schema/src/common/logs.cue [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package common
+
+LogsSortOrder:     "Descending" | "Ascending"                 @cuetsy(kind="enum")
+LogsDedupStrategy: "none" | "exact" | "numbers" | "signature" @cuetsy(kind="enum",memberNames="none|exact|numbers|signature")
```
Comment (ivanahuckova) [inline packages/grafana-schema/src/common/logs.cue:4]: I am curious, why do we set `memberNames` here if they have the same names as enums, but we don't do it for `LogsSortOrder`?

**Review by ivanahuckova — APPROVED** [review]
LGTM! Thanks @Clarity-89 for doing this work for us. 🧡

File: packages/grafana-schema/src/common/logs.cue (hunk 1 above)
Comment (Clarity-89) [reply packages/grafana-schema/src/common/logs.cue:4]: The member names are automatically capitalised in the output TS code, so if we want to preserve their casing we need to define those explicitly. Not necessary for `LogsSortOrder` as those are already capitalised and we want to have them like that in the output.

Comment (ivanahuckova) [reply packages/grafana-schema/src/common/logs.cue:4]: Thank you for explanation. Make sense now. 🙂

**Pull request merged**